scripts/babel/transform-debug-target.js

```diff
@@ -16,15 +15,32 @@ const plugin = ({ template, types: t }) => {
 
   return {
     visitor: {
+      ObjectProperty(path) {
+        if (path.node.key && path.node.key.name === dispatchProperty && !path.node[visited]) {
+          path.node[visited] = true;
+		      path.node.value = t.conditionalExpression(
```

Comment:
Just curious - how come you went from the template syntax to this?

Reply:
templating will always add `semicolons` which isn't desirable for properties

Reply:
there's alternatives in `@babel/template` for this, but I'm fine with getting a working version in first :) https://babeljs.io/docs/en/babel-template#templateexpression

Reply:
Removing the semicolon after NODE in the template string (`NODE;`) doesn't fix that?

Reply:
My template doesn't have a semi https://github.com/FormidableLabs/urql/pull/671/commits/c4e657c025afbe6de6b527942dd7a188086d4357#diff-21f64d2a8eadde32a78a9f45d0587966L9

Reply:
Woah, that's super unusual